modules/apps/client-extension/client-extension-test/src/testIntegration/java/com/liferay/client/extension/type/internal/configuration/test/CETConfigurationFactoryTest.java

```diff
@@ -109,6 +126,118 @@ public void testActivate() throws Exception {
 		}
 	}
 
+	@Test
+	public void testControlPanelThemeCSSCET() throws Exception {
+		String name = RandomTestUtil.randomString();
+
+		Company company = CompanyTestUtil.addCompany();
+
+		String pid1 = ConfigurationTestUtil.createFactoryConfiguration(
+			_PID,
+			HashMapDictionaryBuilder.<String, Object>put(
+				"baseURL", "${portalURL}/o/" + name
+			).put(
+				"companyId", company.getCompanyId()
+			).put(
+				"name", name
+			).put(
+				"projectId", name
+			).put(
+				"projectName", name
+			).put(
+				"service.pid", _PID + "~" + name
+			).put(
+				"type", "themeCSS"
+			).put(
+				"typeSettings", Collections.singletonList("scope=controlPanel")
+			).put(
+				"userId", TestPropsValues.getUserId()
+			).put(
+				"webContextPath", "/" + name
+			).build());
+
+		String pid2 = null;
+
+		try {
+			Layout layout = _getControlPanelLayout(company.getCompanyId());
+
+			Assert.assertNotNull(
+				_clientExtensionEntryRelLocalService.
+					fetchClientExtensionEntryRel(
+						_portal.getClassNameId(Layout.class), layout.getPlid(),
+						ClientExtensionEntryConstants.TYPE_THEME_CSS));
+
+			name = RandomTestUtil.randomString();
+
+			try (LogCapture logCapture = LoggerTestUtil.configureLog4JLogger(
+					"com.liferay.client.extension.type.internal." +
+						"configuration.CETConfigurationFactory",
+					LoggerTestUtil.ERROR)) {
```

Comment:
This try-with-resources statement can just open the whole block, no need to nest `try` statements.